scripts/check-changelog.py

```diff
@@ -32,6 +51,6 @@
     if not any(d in changelog for d in acceptable_dates):
         print((
             "The current date (%s) isn't mentioned in the changelog. "
```

Comment:
I believe that's the pyformat behaviour when you have a single quote inside. I've changed it so we instead escape the ' and pyformat seems happy to accept that.